packages/react-instantsearch/src/components/Highlight.js

```diff
@@ -1,19 +1,6 @@
 import React from 'react';
+import Highlighter from './Highlighter';
 
-export default function Highlight({hit, attributeName, highlight}) {
-  const parsedHighlightedValue = highlight({hit, attributeName});
-  const reactHighlighted = parsedHighlightedValue.map((v, i) => {
-    const key = `split-${i}-${v.value}`;
-    if (!v.isHighlighted) {
-      return <span key={key} className="ais-Highlight__nonHighlighted">{v.value}</span>;
-    }
-    return <em key={key} className="ais-Highlight__highlighted">{v.value}</em>;
-  });
-  return <span className="ais-Highlight">{reactHighlighted}</span>;
+export default function Highlight(props) {
+  return <Highlighter path="_highlightResult" {...props}/>;
 }
-
-Highlight.propTypes = {
```

Comment:
Well if I add them then you get two warnings if a props is missing. From the higlight, and from the highlighter. But I can definitely add them back.